.github/workflows/pr_check_load_test.yml

```diff
@@ -28,32 +23,33 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest]
+        configFile: ['sample-test-url.yaml', 'sample-test-jmx.yaml'] 
+        
     runs-on: ${{ matrix.os }}
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    
-    - uses: actions/checkout@v2
-      name: Checkout from PR branch
-      with: 
-        repository: ${{ github.event.pull_request.head.repo.full_name }}
-        path: 'load-testing'
-        ref: ${{ github.event.pull_request.head.ref }}
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '20'
 
     - name: Installing dependencies and building latest changes
       run: |
-        cd load-testing
-        npm install
-        npm run compile
-    
+        npm install --include=dev -f
+        npm ci
+
     - name: Azure authentication
       uses: azure/login@v1
       with:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
   
     - name: 'Azure Load Testing'
-      uses: ./load-testing/
+      uses: Azure/load-testing@b6e25a93f77ce4988759e3629fe3d641c154d82b
```

Comment:
This should either use the `COMMIT_ID` defined above or the local version similar to how it was earlier

Reply:
its not working either way, ill fix this in followup pr, if i change it then this build will fail because it retriggers.